views/layouts/coursemology.org.html.slim

```diff
@@ -34,13 +34,16 @@ html
               span.icon-bar
           a.navbar-brand href=root_path
             = t('layout.coursemology')
-          div.fb-like[data-href="https://www.facebook.com/coursemology"
-                      data-width="20"
-                      data-layout="button_count"
-                      data-action="like"
-                      data-size="small"
-                      data-show-faces="false"
-                      data-share="false"]
+          / .fb-like-button is our custom class that we use for CSS styling;
+            .fb-like is Facebook's code which is required for its SDK to work
+          div.fb-like-button
```

Comment:
Can I have a request to hide this on mobile screens? It kinds of screws up the navbar (ie. not enough space). Sorry about that @caramelcat 😝 

Can solve it by adding a `.hidden-xs` class to the div.

 <img src="https://cloud.githubusercontent.com/assets/4353853/19099217/e99608c8-8ae6-11e6-97b6-4088560b4634.gif" alt="fb-like" width="320" height="282">